### PR TITLE
Final configuration before release

### PR DESCRIPTION
**Pull request by AnnaritaX**

### CODE REVIEW

File: Heat-Bundle/pom.xml [hunk 1]
```diff
@@ -2,34 +2,28 @@
 <project xmlns="http://maven.apache.org/POM/4.0.0" xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance" xsi:schemaLocation="http://maven.apache.org/POM/4.0.0 http://maven.apache.org/xsd/maven-4.0.0.xsd">
     <modelVersion>4.0.0</modelVersion>
     <parent>
-        <groupId>hcom.funcTest</groupId>
+        <groupId>com.hotels</groupId>
         <artifactId>Heat-Core</artifactId>
```
Comment (massdosage) [inline Heat-Bundle/pom.xml]: All our other projects have followed the convention of the module names being all lower case, so these should become "heat-core", "heat-bundle" etc. I think this is easier for end users as they don't have to think about case and whether the names are case-sensitive or not.

Seeing as we're making so many changes we might as well go full on here ;)

File: Heat-Bundle/pom.xml [hunk 2]
```diff
@@ -2,34 +2,28 @@
...
-        <version>1.0</version>
+        <version>1.0.0</version>
```
Comment (massdosage) [inline Heat-Bundle/pom.xml]: I would have all the versions set to 1.0.0-SNAPSHOT until we're ready to do a release. Should be easy to change them all in one go using the versions plugin.

File: code-of-conduct.md [hunk 3]
```diff
@@ -0,0 +1,73 @@
...
+when an individual is representing the project or its community. Examples of
+representing a project or community include using an official project e-mail
+address, posting via an official social media account, or acting as an appointed
+representative at an online or offline event. Representation of a project may be
+further defined and clarified by project maintainers.
+
+## Enforcement
+
+Instances of abusive, harassing, or otherwise unacceptable behavior may be
+reported by contacting the project team at [INSERT EMAIL ADDRESS]. All
```
Comment (massdosage) [inline code-of-conduct.md]: We need to fill in something here for the email adress, you could leave it like this until we've created a Google Group for the project and come back to it later?

Comment (AnnaritaX) [reply code-of-conduct.md]: yes, let's change it later

File: doc/readme_firstConf.md [hunk 4]
```diff
@@ -40,7 +40,7 @@ We have to customise the dependency section and the properties one.
     <dependencies>
         <dependency>
             <groupId>hcom.funcTest</groupId>
-            <artifactId>Heat-Bundle</artifactId>
+            <artifactId>heat-bundle</artifactId>
             <version>1.0</version>
```
Comment (massdosage) [inline doc/readme_firstConf.md]: 1.0.x to match the other documentation

File: doc/readme_firstConf.md [hunk 5]
```diff
@@ -40,7 +40,7 @@ We have to customise the dependency section and the properties one.
     <dependencies>
         <dependency>
             <groupId>hcom.funcTest</groupId>
```
Comment (massdosage) [inline doc/readme_firstConf.md]: Doesn't this need to be com.hotels?

File: doc/readme_extmodule.md [hunk 6]
```diff
@@ -26,21 +26,21 @@ Therefore, besides the core placeholders (explained in the [placeholder section]
...
-Anyway, if you want use the minimal set of dependencies to define your own Placeholder, you could use the following subset of dependencies as an alternative to "Heat-Bundle":
+Usually, the only suggested dependency to use HEAT is "heat-bundle", that includes all the useful dependencies for use or extend the framework. 
+Anyway, if you want use the minimal set of dependencies to define your own Placeholder, you could use the following subset of dependencies as an alternative to "heat-bundle":
 
 ```xml
 <!-- my pom.xml --> 
 ...
     <dependencies>
         <dependency>
             <groupId>hcom.funcTest</groupId>
```
Comment (massdosage) [inline doc/readme_extmodule.md]: com.hotels?

File: doc/readme_extmodule.md [hunk 7]
```diff
@@ -26,21 +26,21 @@ Therefore, besides the core placeholders (explained in the [placeholder section]
...
-            <artifactId>Heat-ModuleSupport</artifactId>
+            <artifactId>heat-module-support</artifactId>
             <version>1.0.X</version>
         </dependency>
         <dependency>
             <groupId>hcom.funcTest</groupId>
```
Comment (massdosage) [inline doc/readme_extmodule.md]: com.hotels?

**Pull request merged**